components/omega/doc/design/OmegaV0ShallowWater.md

```diff
@@ -0,0 +1,692 @@
+
+## 3 Algorithmic Formulation
+
+
+### 3.1 Governing Equations
+
+#### 3.1.1 Continuous Equations
+
+The algorithms for Omega-0 are given here in full detail, with all variables defined in section 3.2 below.
+We begin with the standard shallow water equations, which are derived from conservation of momentum and mass in a single layer with uniform density in a rotating frame. The standard presentation may be found in standard textbooks on Geophysical Fluid Dynamics, such as those by Vallis (2017), Cushman‐Roisin and Beckers (2011), Pedlosky (1987), and Gill (2016). See references at bottom. In continuous form, these are
+
+$$
+\frac{\partial \boldsymbol{u}}{\partial t} + \left(\boldsymbol{u}\cdot\nabla\right)\boldsymbol{u}  + f \boldsymbol{k}\times\boldsymbol{u}
+= -g\nabla(h-b)
+\hspace{1cm}
+$$
+
+$$
+\frac{\partial h}{\partial t} + \nabla \cdot \left(h \boldsymbol{u}\right) = 0,
+\hspace{1cm}
+$$
+
+Next we replace the non-linear advection term with the right-hand side of the vector identity
+$$
+\boldsymbol{u} \cdot \nabla \boldsymbol{u} &= (\nabla \times \boldsymbol{u}) \times \boldsymbol{u} + \nabla \frac{|\boldsymbol{u}|^2}{2} \\
+&= \{\boldsymbol{k} \cdot (\nabla \times \boldsymbol{u})\} \boldsymbol{k} \times \boldsymbol{u} + \nabla \frac{|\boldsymbol{u}|^2}{2} \\
+&= \omega \boldsymbol{u}^{\perp} + \nabla K.
+$$
+
+The governing equations for Omega-0 in continuous form are then
+$$
+\frac{\partial \boldsymbol{u}}{\partial t} + q\left(h\boldsymbol{u}^{\perp}\right) = -g\nabla(h-b) - \nabla K + \nu_2 \nabla^2 \boldsymbol{u} - \nu_4 \nabla^4 \boldsymbol{u} + \mathcal{D} + \mathcal{F}
+\hspace{1cm}   (1)
+$$
+
+$$
+\frac{\partial h}{\partial t} + \nabla \cdot \left(h \boldsymbol{u}\right) = 0,
+\hspace{1cm}   (2)
+$$
+
+$$
+\frac{\partial h \phi}{\partial t} + \nabla \cdot \left(h \boldsymbol{u} \phi\right) = \kappa_2 h \nabla^2 \phi - \kappa_4 h \nabla^4 \phi.
+\hspace{1cm}   (3)
+$$
+These are typically referred to as the momentum equation (or velocity equation), the thickness equation, and the tracer equation. 
+
+The additional momentum terms in (1) are viscous dissipation (del2 and del4), and drag and forcing are as follows:
+$$
+\mathcal{D} = -Ra \, \boldsymbol{u}-C_D \frac{\boldsymbol{u}\left|\boldsymbol{u}\right|}{h}
+$$
+
+$$
+\mathcal{F} = - C_W \frac{(\boldsymbol{u}_W - \boldsymbol{u})\left|\boldsymbol{u}_W - \boldsymbol{u}\right|}{h}
+$$
+The drag consists of simple Rayleigh drag for spin-up as well as quadratic bottom drag. The forcing is a quadratic wind forcing (see equation 1 in [Lilly et al. 2023](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2022MS003327)).
```

Comment:
the wind forcing seems out of place here.  The form seems fine since it isn't unlike the coupled form, but I'd suggest putting a more generic form like `tau/(rho*h` approach

Reply:
I don't understand. Dimensionally, the coefficient of a quadratic velocity (normalized by thickness) must be non-dimensional. So the C_D and C_W are non-dimensional. Also, there is no density in a shallow water equation set. 

I was just adding some forcing and dampening terms that would transfer directly to Omega-1. 

FYI, there is a [Stommel basin test case](https://gmd.copernicus.org/articles/16/1297/2023/#section8) with an exact equilibrium solution that includes wind forcing for a shallow water test case. We listed that test as a lower-priority additional test on the [performance testing design document](https://portal.nersc.gov/project/e3sm/mpeterse/omega/design-omega0-sw/html/design/OmegaV0ShallowWater.html#inertia-gravity-wave-linearized-shallow-water-no-tracers). I'm sure we could convert that forcing function to whatever form of this coefficient we choose.

Reply:
I guess all I'm saying is that in the barotropic continuous equations the stress would be written as `tau/(rho_w*h)` and what you write in here is an assumed form of tau/rho_w, my suggestion was for the full continuous equation to leave this as `tau/h` and then in the later section specify the quadratic assumption.  Not a big deal to leave as is though.